gitlab/docs/runner/gauge.go_memstats_next_gc_bytes.md

```diff
@@ -0,0 +1,8 @@
+---
+title: go_memstats_next_gc_bytes
+brief: Number of heap bytes when next garbage collection will take place.
```

Comment:
Another general comment - Brief descriptions can be more brief in some places. They show up in a table in the product docs, so really long ones will make the table a lot bigger. Not a big thing, just if you have time.